ckanext/pages/actions.py

```diff
@@ -110,7 +112,7 @@ def _pages_list(context, data_dict):
         pg_row = {'title': pg.title,
                   'content': pg.content,
                   'name': pg.name,
-                  'publish_date': pg.publish_date,
+                  'publish_date': pg.publish_date.isoformat() if pg.publish_date else None,
```

Comment:
No, I don't think it affects controller's use of this function. I tested a few ckanext_pages_list API calls against an instance that has pages with and without publish_date, they all worked and displayed correct information.
Without the change, the API call will break if the page actually has a publish_date.